examples/QIR/Development/DebugInfo/CoreTypes.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
These files are here so that the development QIR can easily be compiled to an executable. However, they are library files that would need updating when they change. Is there a better way to make the development process easier for the user without including all these files? I did at least put a note in the README that these files may need to be updated.

Reply:
These files are included as part of the QIR runtime package. In order to compile the executable, the library files should have been pulled from somewhere and it is likely these headers can be pulled from the same place.

Reply:
You're right, these are all in the build folder so I'll adjust accordingly